wagtail/admin/wagtail_hooks.py

```diff
@@ -917,7 +909,7 @@ def register_aging_pages_report_menu_item():
 
 @hooks.register("register_admin_menu_item")
 def register_reports_menu():
-    return ReportsMenuItem(_("Reports"), reports_menu, icon_name="site", order=9000)
+    return SubmenuMenuItem(_("Reports"), reports_menu, icon_name="site", order=9000)
```

Comment:
I’m uncertain of this. Did I misunderstand some interest of having a separate class for `ReportsMenuItem`?

Reply:
Looks like this was introduced in https://github.com/wagtail/wagtail/commit/0db98757b41845d0f8ec631042f4cf0dc39186c3. It used to override the `is_shown` method but that override has since been removed.

I think this is fine to remove

Reply:
Thank you, should’ve known to check the blames